cmd/fyne/internal/commands/release.go

```diff
@@ -447,6 +443,26 @@ func (r *Releaser) validate() error {
 	return nil
 }
 
+func (r *Releaser) writeEntitlements(tmpl *template.Template, entitlementData interface{}) (cleanup func(), err error) {
+	entitlementPath := filepath.Join(r.dir, "entitlements.plist")
+	entitlements, err := os.Create(entitlementPath)
+	if err != nil {
+		return nil, err
+	}
+	defer func() {
+		if r := entitlements.Close(); r != nil {
+			err = r
+		}
+	}()
+
+	if err := tmpl.Execute(entitlements, entitlementData); err != nil {
+		return nil, err
+	}
+	return func() {
+		_ = os.Remove(entitlementPath)
```

Comment:
If error is not handled why not just:
```go
os.Remove(entitlementPath)
```

Reply:
because that is a sign to your future self, other developers (and linters, if you use them) that you are well aware of the fact that you are ignoring the error. If you use `os.Remove(entitlementPath)` nobody knows if you just forgot to handle the error or you purposefully ignore it.

Reply:
Good point :)